src/main/java/com/epam/reportportal/extension/azure/AzureExtension.java

```diff
@@ -184,4 +224,165 @@ private void removeListeners() {
 		pluginCommandMapping.put("testConnection", new TestConnectionCommand());
 		return pluginCommandMapping;
 	}
+
+	@Override
+    // Never called method. Connection is tested via the command.
+	public boolean testConnection(Integration system) {
+		return false;
+	}
+
+	@Override
+    // TODO: Implement in the future
+	public Optional<Ticket> getTicket(String id, Integration system) {
+		return Optional.empty();
+	}
+
+    @Override
+    // TODO: Implement in the future
+    public Ticket submitTicket(PostTicketRQ ticketRQ, Integration system) {
+        return null;
+    }
+
+	@Override
+	public List<PostFormField> getTicketFields(String issueType, Integration integration) {
+		Map<String, Object> params = integration.getParams().getParams();
+		String organizationUrl = params.get("url").toString();
+		String projectName = params.get("project").toString();
+		String personalAccessToken = params.get("oauthAccessKey").toString();
+
+		ApiClient defaultClient = getConfiguredApiClient(personalAccessToken);
+		String organizationName = organizationUrl.replace(defaultClient.getBasePath(), "");
+
+		ClassificationNodesApi nodesApi = new ClassificationNodesApi(defaultClient);
+		List<WorkItemClassificationNode> areaNodes = new ArrayList<>();
+		List<WorkItemClassificationNode> iterationNodes = new ArrayList<>();
+		try {
+            List<WorkItemClassificationNode> nodes = nodesApi
+                    .classificationNodesGetRootNodes(organizationName, projectName, API_VERSION, DEPTH);
+			for (WorkItemClassificationNode node : nodes) {
+				if (node.getStructureType().equals("area")) {
+					areaNodes = extractNestedNodes(node);
+				} else if (node.getStructureType().equals("iteration")) {
+					iterationNodes = extractNestedNodes(node);
+				}
+			}
+		} catch (ApiException e) {
+            LOGGER.error("Unable to load classification nodes: " + e.getMessage(), e);
+			throw new ReportPortalException(ErrorType.UNABLE_INTERACT_WITH_INTEGRATION,
+					"Unable to load classification nodes. Code: " + e.getCode() + ", Message: " + e.getMessage(), e);
+		}
+
+		WorkItemTypesFieldApi issueTypeFieldsApi = new WorkItemTypesFieldApi(defaultClient);
+		FieldsApi fieldsApi = new FieldsApi(defaultClient);
+        List<PostFormField> ticketFields = new ArrayList<>();
+		try {
+            List<WorkItemTypeFieldWithReferences> issueTypeFields = issueTypeFieldsApi
+                    .workItemTypesFieldList(organizationName, projectName, issueType, API_VERSION, EXPAND);
+
+            for (WorkItemTypeFieldWithReferences field : issueTypeFields) {
+				WorkItemField detailedField = null;
+                // Skip fields that return 404 on request
+				try {
+					detailedField = fieldsApi
+							.fieldsGet(organizationName, field.getReferenceName(), projectName, API_VERSION);
+				} catch (ApiException e) {
```

Comment:
Check it's 404 error here. ApiException should have the code.

Reply:
Added check